content/docs/user-guide/how-to/add-output-to-stage.md

```diff
@@ -0,0 +1,50 @@
+# Add Output to Stage
+
+There are situations where we run a stage without specifying all the
+<abbr>outputs</abbr>, and then later want to add more outputs to that stage. If
+we use `dvc run` to add outputs, it will re-run the stage.
+
+Running a stage is an expensive step, as it requires time and resources to
+execute the command. Follow the steps below to add an output to the stage
+without running it again.
+
+We have an example stage `prepare` defined in `dvc.yaml`. To add another output
+`data/test` to this stage, we can edit `dvc.yaml`. This is how it looks after
+the change:
+
+```dvc
+$ cat dvc.yaml
+stages:
+  prepare:
+    cmd: python src/prepare.py
+    deps:
+    - src/prepare.py
+    outs:
+    - data/train
++   - data/test
```

Comment:
We have to decide whether we're using ` ```dvc ` and the `$ cat ...` console example OR the ` ```git ` format (no `$` line) which supports the `+` line highlighting. The latter seems more important it has already been mentioned that this is a dvc.yaml file. BTW I think all lines need an extra space at the beginning to align with the one starting with `+`, please check how it gets rendered in your local dev env.

Reply:
Also data/test -> data/validate please